pkg/block/metadata/meta.go

```diff
@@ -51,8 +51,9 @@ type Meta struct {
 
 // Thanos holds block meta information specific to Thanos.
 type Thanos struct {
-	Labels     map[string]string `json:"labels"`
-	Downsample ThanosDownsample  `json:"downsample"`
+	Labels      map[string]string `json:"labels"`
+	Downsample  ThanosDownsample  `json:"downsample"`
+	SizeInBytes *uint64           `json:"size"`
```

Comment:
I guess this **does not** include meta file?

Reply:
Worth to comment.

Reply:
Indeed, will do.